lib/openrtb2_3/bidResponse.js

```diff
@@ -83,13 +72,14 @@ BidResponseBuilder.prototype.build = function() {
               this._id
             ).removeUndefined();
 
-        var validationErrors = this.validateBidResponse(bidResponse);
-        resolve({
-         bidResponse: bidResponse,
-         meta: {
-          validationErrors: validationErrors
-         }  
-        });
+        var validationResultArray = validator.validate(bidResponseSchema, bidResponse);
+
+        if(validationResultArray.length === 0){
+          resolve(bidResponse);
+        } else {
+          reject(new ValidationError(validationResultArray));
```

Comment:
I think you shouldn't pass an array as a message. Instead you can create a string or pass an object in the constructor containing both the message and a validtionErrors array which can then be inspected by the user.